core/src/main/java/org/frankframework/compression/ZipWriter.java

```diff
@@ -110,21 +107,20 @@ public MessageZipEntry createPart(Message input, PipeLineSession session, Parame
 
 	@Override
 	public Message build(List<MessageZipEntry> parts) throws IOException {
-		File file;
```

Comment:
1. These changes remove functionality (to return a filename instead of the file). Probably deprecated, but then the deprecation should be done in 9.0, not 8.3
2. Currently these changes also break 4 tests in IAF-Test that expect the file to exist on disk at some point.

Reply:
I've kept all functionality as-is. Nothing has been removed or is broken.

Reply:
It looked to me as if some functionality regarding returning filename or using filename was gone. I may have misinterpreted the code. Looking at the failures in IAF-Test on a missing file that was expected.